payjoin/src/core/send/mod.rs

```diff
@@ -718,10 +718,6 @@ impl V1Context {
     /// valid you will get appropriate PSBT that you should sign and broadcast.
     #[inline]
     pub fn process_response(self, response: &[u8]) -> Result<Psbt, ResponseError> {
-        if response.len() > MAX_CONTENT_LENGTH {
-            return Err(ResponseError::from(InternalValidationError::ContentTooLarge));
-        }
```

Comment:
@nothingmuch what's the rationale for making the client do this? Just because we're not taking a `Read` anymore it doesn't make sense to do here?